src/mixins/amalgamation-mixin.ts

```diff
@@ -88,7 +88,7 @@ export default class AmalgamationMixin extends Vue {
     if (!this.isRoleStaff && business.type === AmlTypes.FOREIGN) {
       return AmlStatuses.ERROR_FOREIGN
     }
-    if (!this.isRoleStaff && business.type === AmlTypes.LEAR && business.legalType === CorpTypeCd.EXTRA_PRO_A) {
+    if (!this.isRoleStaff && business.type === AmlTypes.FOREIGN && business.corpNumber[0] === CorpTypeCd.EXTRA_PRO_A) {
```

Comment:
I know it's a bit funky (`business.corpNumber[0]`), but since we're not storing the legalType, had to do this. We're not storing the legal Type for Foreign businesses (A are now treated as foreign).

Reply:
Also, I tested for regular user:
![only staff](https://github.com/bcgov/business-create-ui/assets/122301442/44f9142d-abb9-4ee4-bf40-fe6b413b0a83)

Reply:
Why not look at the legalType that comes back from the business lookup?

Eg,
![image](https://github.com/bcgov/business-create-ui/assets/10002889/f8871311-7a34-4969-9673-3c516eccfab2)

Reply:
Oh, because it's not in the Foreign object that we saved...

Reply:
Actually, why do you even need this? Won't A companies, which are tagged as Foreign, match line 88?

Reply:
You're absolutely RIGHT. Removing it.